packages/antd/src/templates/ObjectFieldTemplate/index.js

```diff
@@ -62,12 +61,12 @@ const ObjectFieldTemplate = ({
         ? 24
         : 12;
 
-    if (_.isObject(colSpan)) {
+    if (colSpan instanceof Object) {
```

Comment:
I'm pretty sure this isn't a 1:1 change given that `isObject` is implemented as:
```
function isObject(value) {
  var type = typeof value;
  return value != null && (type == 'object' || type == 'function');
}
```